# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/audioipc/audioipc/Cargo.toml
# Commit: 0a7873a6b522
# Full Hash: 0a7873a6b522ce4aef9a32ae4a47f093ba33e0f4
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2020-01-26 09:48:16
# Regressor Bug: 1590249
# File Overlap Count: 3
# Description:
#   Bug 1590249 - Update audioipc to 86d49ddf.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60615
# ==============================================================================

diff -r 36f6505b50c9 -r 0a7873a6b522 media/audioipc/audioipc/Cargo.toml
--- a/media/audioipc/audioipc/Cargo.toml	Sat Jan 25 21:30:14 2020 +0000
+++ b/media/audioipc/audioipc/Cargo.toml	Sat Jan 25 21:31:32 2020 +0000
@@ -19,7 +19,7 @@
 serde_derive = "1.*.*"
 tokio = "0.1"
 tokio-io = "0.1"
-audio_thread_priority = "0.20.2"
+audio_thread_priority = "0.21"
 
 [target.'cfg(unix)'.dependencies]
 iovec = "0.1"
@@ -29,9 +29,9 @@
 tokio-reactor = "0.1"
 
 [target.'cfg(windows)'.dependencies]
+mio = "0.6.19"
 miow = "0.3.3"
 mio-named-pipes = { git = "https://github.com/alexcrichton/mio-named-pipes" }
-tokio-named-pipes = { git = "https://github.com/NikVolf/tokio-named-pipes", branch = "stable" }
 winapi = { version = "0.3.6", features = ["combaseapi", "objbase"] }
 
 [dependencies.error-chain]